jax_galsim/core/wrap_image.py

```diff
@@ -33,3 +33,107 @@ def _body_i(i, vals):
 
     im = jax.lax.fori_loop(0, im.shape[0], _body_i, im)
     return im
+
+
+@jax.jit
+def expand_hermitian_x(im):
+    return jnp.concatenate([im[:, 1:][::-1, ::-1].conjugate(), im], axis=1)
+
+
+@jax.jit
+def contract_hermitian_x(im):
+    return im[:, im.shape[1] // 2 :]
+
+
+@jax.jit
+def wrap_hermitian_x(im, im_xmin, im_ymin, wrap_xmin, wrap_ymin, wrap_nx, wrap_ny):
+    im_exp = expand_hermitian_x(im)
+    im_exp = wrap_nonhermitian(
+        im_exp, wrap_xmin - im_xmin, wrap_ymin - im_ymin, wrap_nx, wrap_ny
+    )
+    return contract_hermitian_x(im_exp)
+
+
+@jax.jit
+def expand_hermitian_y(im):
+    return jnp.concatenate([im[1:, :][::-1, ::-1].conjugate(), im], axis=0)
+
+
+@jax.jit
+def contract_hermitian_y(im):
+    return im[im.shape[0] // 2 :, :]
+
+
+@jax.jit
+def wrap_hermitian_y(im, im_xmin, im_ymin, wrap_xmin, wrap_ymin, wrap_nx, wrap_ny):
+    im_exp = expand_hermitian_y(im)
+    im_exp = wrap_nonhermitian(
+        im_exp, wrap_xmin - im_xmin, wrap_ymin - im_ymin, wrap_nx, wrap_ny
+    )
+    return contract_hermitian_y(im_exp)
+
+
+# I am leaving this code here for posterity. It has a bug that I cannot find.
```

Comment:
@rmjarvis @jecampagne If any of you have a chance to look for bugs in the code below, that'd be cool. No worries if not!